scipy/sparse/sparsetools.py

```diff
@@ -32,22 +32,22 @@ def _deprecated():
 # of rows.
 
 def csr_matvec(N, M, intptr, indices, data, x, y):
-    _sparsetools.csr_matvec(N, M, indptr, indices, data, 1, x, 0, y)
+    _sparsetools.csr_matvec(N, M, indptr, indices, data, 1, x, 1, y)
```

Comment:
I remember a comment was made on the PR #5775 regarding this. I can change it if needed.